apps/web/src/pages/settings/tabs/components/Security.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, colors, Switch, Text } from '../../../../design-system';
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { Description } from '../InAppCenterCard';
+import { ColorScheme, useMantineTheme } from '@mantine/core';
+
+export const Security = () => {
+  const [isHmacEnabled, setIsHmacEnabled] = useState<boolean>(false);
+  const { colorScheme } = useMantineTheme();
+
+  function handlerSwitchChange() {
+    setIsHmacEnabled((prev) => !prev);
+  }
+
+  return (
+    <>
+      <Title>Security</Title>
+      <SubTitle weight="bold">Enable HMAC encryption</SubTitle>
+      <InlineDiv>
+        <Description>
+          HMAC used to verify that the request performed by the specific user. Read more{' '}
+          <StyledHref
+            colorScheme={colorScheme}
+            className={'security-doc-href'}
+            href="https://docs.novu.co/docs/notification-center/getting-started"
+            target="_blank"
+            rel="noreferrer"
+          >
+            here.
+          </StyledHref>
+        </Description>
+      </InlineDiv>
+
+      <RowDiv>
+        <Switch checked={isHmacEnabled} data-test-id="is_active_id" onChange={handlerSwitchChange} />
+        <StyledText>{isHmacEnabled ? 'Enabled' : 'Disabled'}</StyledText>
```

Comment:
@djabarovgeorge could you take a look at the styling of the "Active" mode of the switch in the template editor and just reuse the same component here?

Reply:
Done